### PR TITLE
Feature/cardio intervention

### PR DESCRIPTION
**Pull request by catherineojohnson**

### CODE REVIEW

**Review by ndecleene — APPROVED** [review]

File: docs/source/intervention_models/crm_mgmt/vivarium_outreach.rst [hunk 1]
```diff
@@ -0,0 +1,124 @@
+Vivarium Modeling Strategy - Outreach Intervention 
+--------------------------------------------------
+
+.. todo::
+
+  Add an overview of the Vivarium modeling section.
+
+Decision tree and algorithm for outreach intervention
+
+Visit type: emergency
```
Comment (aflaxman) [inline docs/source/intervention_models/crm_mgmt/vivarium_outreach.rst:10]: I don't think of these visit types as part of the outreach intervention;  I recommend you have a `healtcare_access.rst` to present this information, and then reference the healthcare_access.rst in this doc as relevant.

File: docs/source/intervention_models/crm_mgmt/vivarium_outreach.rst [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+Visit type: screening
+
+.. image:: decision_tree_outreach_screening.svg
+
+Visit type: none
+
+.. image:: decision_tree_outreach_none.svg
+
+Blood pressure ramp: initial prescription
```
Comment (aflaxman) [inline docs/source/intervention_models/crm_mgmt/vivarium_outreach.rst:26]: I recommend that the blood pressure ramp be in a hypertension_treatment.rst file, with sections on the treatment ramp, the initialization strategy, and the effect of the treatment on the SBP risk exposure (for each ramp position)

File: docs/source/intervention_models/crm_mgmt/vivarium_polypill.rst [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Vivarium Modeling Strategy - Polypill Intervention 
+--------------------------------------------------
+
+.. todo::
+
+  Add an overview of the Vivarium modeling section.
+
+Decision tree and algorithm for polypill intervention
```
Comment (aflaxman) [inline docs/source/intervention_models/crm_mgmt/vivarium_polypill.rst:8]: I expect this to reference the hypertension treatment doc and describe how some ramp positions can be achieved by a polypill; it should say what the effect of polypill is on adherence, and who gets polypill (perhaps with reference to the scenarios.rst description of what the coverage of polypill is in your alternative scenarios.)

File: docs/source/intervention_models/crm_mgmt/vivarium_lifestyle.rst [hunk 4]
```diff
@@ -0,0 +1,8 @@
+Vivarium Modeling Strategy - Lifestyle Intervention
+---------------------------------------------------
+
+.. todo::
+
+  Add an overview of the Vivarium modeling section.
+
+Decision tree and algorithm for lifestyle intervention
```
Comment (aflaxman) [inline docs/source/intervention_models/crm_mgmt/vivarium_lifestyle.rst:8]: see my comments on the vivarium_polypill.rst, I expect this doc to hit similar points relevant to lifestyle intervention.

File: docs/source/intervention_models/crm_mgmt/scenarios.rst [hunk 5]
```diff
@@ -0,0 +1,64 @@
+Scenarios
+*********
+
+Outreach
+~~~~~~~~~~~~~~
+Under this scenario, the effectiveness of the delivery of SBP-lowering and LDL-c lowering therapies would increase through methods such as a phone call, app, or support clinic.
```
Comment (aflaxman) [inline docs/source/intervention_models/crm_mgmt/scenarios.rst:6]: as we discussed today, I would like the scenarios.rst to describe what the coverage of the 3 interventions (outreach, lifestyle, polypill) is at each time in each scenario.  A lot of the background info about each specific intervention can go into the corresponding intervention doc.

File: docs/source/intervention_models/crm_mgmt/scenarios.rst [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+[Metz-et-al-2000]_ 
+
+**Scenarios:**
+	- Lifestyle 1.0: Enrollment will occur following a routine health facility or primary care visit. Individuals with BMI >= 25 or FPG 100-125 mg/dl at the time of the visit will receive weekly sessions for 6 months followed by monthly sessions for 6 months. Each session attended will result in 0.3% loss of body weight (initially represented as BMI, but eventually to use weight when available). Adherence will decline in a linear fashion, with 50% of individuals continuing sessions at 6 months and all completing sessions by 1 year.  
+	- Lifestyle 0.5: Scenario described above but with 50% of adherence at initiation; adherence will decline in a linear fashion from that point. 
+
+
+4.1.4 Combined scenarios
+~~~~~~~~~~~~~~~~~~~~~~~~
+	- Combination 1.0: Combination of outreach 1.0, polypill 1.0, and lifestyle 1.0
```
Comment (aflaxman) [inline docs/source/intervention_models/crm_mgmt/scenarios.rst:63]: specify when these levels of coverage should be attained, and if there should be an immediate scale up or a gradual scale up from baseline (and if gradual, describe how, e.g. linear scale up from baseline levels on 2022-01-01 to 100% coverage on 2023-01-01)

**Review by aflaxman — APPROVED** [review]

File: docs/source/intervention_models/crm_mgmt/vivarium_outreach.rst (hunk 1 above)
Comment (catherineojohnson) [reply docs/source/intervention_models/crm_mgmt/vivarium_outreach.rst:10]: Added visit.rst; linked in the vivarium_{intervention} sections

File: docs/source/intervention_models/crm_mgmt/vivarium_outreach.rst (hunk 2 above)
Comment (catherineojohnson) [reply docs/source/intervention_models/crm_mgmt/vivarium_outreach.rst:26]: have created separate file; not all details have been added. file is linked to vivarium_{intervention} docs.

File: docs/source/intervention_models/crm_mgmt/scenarios.rst (hunk 5 above)
Comment (catherineojohnson) [reply docs/source/intervention_models/crm_mgmt/scenarios.rst:6]: have made edits to the outreach section; will follow up with additional edits to the other two interventions.

File: docs/source/intervention_models/crm_mgmt/scenarios.rst (hunk 6 above)
Comment (catherineojohnson) [reply docs/source/intervention_models/crm_mgmt/scenarios.rst:63]: have added to the outreach section; will add to the other two sections when I flesh them out more.

**Pull request merged**